nuxt.config.js

```diff
@@ -2,6 +2,10 @@ export default {
   target: 'static',
   ssr: true,
   head: {
+    htmlAttrs: {
+      lang: 'fa',
```

Comment:
this will be added to every single language if I am not mistaken so we need to add rtl only if the lang is fa.

Reply:
So how do you suggest to do it? What's the proper way to do so? Is there a way to get the current language and act according to that?

Reply:
good question. we will have to try something using if i18n.locale === fa add rtl. Never tried it before but give it a go and if you get stuck just comment it out and we will look into it next week.

Reply:
I tried this and it is working perfectly:

```javascript
    htmlAttrs: {
      lang: process.env.NUXT_LOCALE,
      dir: ['fa', 'ar', 'he'].includes(process.env.NUXT_LOCALE) ? 'rtl' : 'ltr'
    }
```

And for the production we can for sure generate the website using the local wanted.

The only three languages I know which are RTL are Persian, Arabic and Hebrew so I added them to the array.

Reply:
fantastic. great job 💪